### DISCUSSION TITLE
Create github page for monaco

### DISCUSSION THREAD

**Discussion by patrickpichler** [Dynatrace/dynatrace-configuration-as-code#135] [Archive] [answered]
We could create a [github page](https://pages.github.com/) for monaco hosting the documentation
of the latest release. 

It could also contain download links and could act as the main thing a user interacts with, instead 
of the github repo directly.

**Comment by didiladi** [comment, marked as answer]
I like the idea. Are there some nice tools for autogenerating the documentation from the readme? :thinking:

**Reply by patrickpichler** [reply]
All we would need is to render the README as html and show it on the web page. 

I had something the way zig does it in mind: https://ziglang.org/
When you press on documentation you see the rendered markdown.

**Reply by UnseenWizzard** [reply]
afaik github pages default is to use jekyll to generate static pages (from e.g. markdown) https://jekyllrb.com/

my website is a github page using jekyll and a slightly modified theme https://github.com/UnseenWizzard/unseenwizzard.github.io

**Reply by UnseenWizzard** [reply]
so it might be possible to just throw the readme markdown in

**Reply by patrickpichler** [reply]
That would even be better :+1: 

So on a release we should create a new sub-folder in the gh-pages branch and link to it.

**Reply by CruzanCaramele** [reply]
Creating a GiBook for the documentation

**Reply by tatomir146** [reply]
@CruzanCaramele what's the current status here? Please share the ticket here and move this discussion to Archive.

**Reply by CruzanCaramele** [reply]
https://dynatrace-oss.github.io/dynatrace-monitoring-as-code/next

**Comment by tatomir146** [comment]
Solved: https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/pull/344
